Hypnotic-Herons/Sprint-2
# created department template

# Description of Proposed Changes
added template for departments

## Steps to Test
Outline the steps to test
```
git fetch --all
git checkout js_departmenttemplate
code .
```
python manage.py makemigrations HRIT
python manage.py migrate
python manage.py createsuperuser (this is just for your local use)
login on the admin view http://127.0.0.1:8000/admin
add a few departments
look at the browser view for computers to see if they are there http://127.0.0.1:8000/HRIT/departments/
you should see the departments in a list
